config/config.go

```diff
@@ -0,0 +1,16 @@
+package config
+
+type Config struct {
+	Taskcluster struct {
+		ClientId    string
+		AccessToken string
+		Certificate string
+	}
+	ProvisionerId string
+	WorkerGroup   string
+	WorkerId      string
+	Capacity      int
+	QueueService  struct {
+		ExpirationOffset int
+	}
+}
```

Comment:
I added a comment at the base of this file (in the PR) indicating that this was just mocking out some config stuff until we had it officially in place.  Pete has been working on his jsonschemago library so we can specify the schemas (in yaml) for the configuration pieces that the worker, engines, and plugins care about and generate a complete schema along with the types for it.